backend/resources/gpml/config.edn

```diff
@@ -223,16 +223,36 @@
                              :app-domain #duct/env "APP_DOMAIN"}}
   :gpml.handler.event/post-params {}
 
-  :gpml.handler.resource/post {:db #ig/ref :duct.database/sql}
+  :gpml.handler.resource/post {:db #ig/ref :duct.database/sql
+                               :mailjet-config
```

Comment:
it feels *strange* that I need to repeat the same configuration several times. Is not the case we want some key to hold that config and pass it around? 🤔 https://github.com/duct-framework/duct/wiki/Configuration#constants
e.g.

```
{[:duct/const :mailjet-config] {:api-key #duct/env "MAILJET_API_KEY"
                                :secret-key #duct/env "MAILJET_SECRET_KEY"
                                :app-name #duct/env "APP_NAME"
                                :app-domain #duct/env "APP_DOMAIN"}]
,,,
:gpml.handler.resource/post {:db #ig/ref :duct.database/sql
                             :mailjet-config #ig/ref :mailjet-config}
,,,
}
I have not tested it
```

Reply:
Ah, yes, I was meaning to look into it, or ask you about it, but it skipped my mind. Thanks for the link, I'll look into this. :)